--- a/CMakeLists.txt	2024-06-27 01:02:44.827003002 +0200
+++ b/CMakeLists.txt	2024-06-27 01:11:49.941487593 +0200
@@ -93,7 +93,7 @@
 ADD_EXECUTABLE(pktinfo src/pktinfo.c src/dupe.c src/pktread.c src/fcommon.c src/global.c)
 ADD_EXECUTABLE(tpkt src/tpkt.c src/dupe.c src/pktread.c src/fcommon.c src/global.c src/pktwrite.c)
 ADD_EXECUTABLE(txt2pkt src/txt2pkt.c src/dupe.c src/pktread.c src/fcommon.c src/global.c src/pktwrite.c)
-ADD_EXECUTABLE(hptlink src/hptlink.c src/global.c)
+ADD_EXECUTABLE(hptlink src/hptlink.c)
 ADD_EXECUTABLE(hpttree src/hpttree.c)
 ADD_EXECUTABLE(hpt src/hpt.c src/dupe.c src/pktread.c src/fcommon.c src/global.c src/pktwrite.c src/toss.c src/scan.c src/scanarea.c src/post.c src/hptafix.c src/seenby.c src/link.c src/carbon.c ${perl_SRC})
 target_link_libraries(pktinfo ${fidoconfig_LIB} ${smapi_LIB} ${husky_LIB})
