diff -urp ftrack-1.16-old/MakeFiles/linux/Makefile ftrack-1.16/MakeFiles/linux/Makefile
--- ftrack-1.16-old/MakeFiles/linux/Makefile	2009-10-28 09:45:47.000000000 +0100
+++ ftrack-1.16/MakeFiles/linux/Makefile	2010-02-22 04:06:16.000000000 +0100
@@ -1,10 +1,10 @@
 # $Id: Makefile 57 2009-10-23 13:46:06Z dukelsky $
-CC	= g++
-CFLAGS	= -Wall -fno-rtti -I../../hpp -I../../src -O3
-CDEFS	= -Dunix -D__linux__ -DHAVE_UNISTD_H -DHAVE_FCNTL_H -DHAVE_MALLOC_H \
-          -DHAVE_STRING_H -DHAVE_TIME_H -DHAVE_SYS_TIME_H -DHAVE_UTIME_H \
-	  -DHAVE_LOCALE_H
-LFLAGS	= -s -L$(SRCDIR)smapi -lsmapilnx
+#CC	= g++
+#CFLAGS	= -Wall -fno-rtti -I../../hpp -I../../src -O3
+#CDEFS	= -Dunix -D__linux__ -DHAVE_UNISTD_H -DHAVE_FCNTL_H -DHAVE_MALLOC_H \
+#          -DHAVE_STRING_H -DHAVE_TIME_H -DHAVE_SYS_TIME_H -DHAVE_UTIME_H \
+#	  -DHAVE_LOCALE_H
+#LFLAGS	= -s -L$(SRCDIR)smapi -lsmapilnx
 SRCDIR	= ../../src/
 #CONFIG	= /etc/ftn/rntrack.cfg
 
diff -urp ftrack-1.16-old/src/Makefile.am ftrack-1.16/src/Makefile.am
--- ftrack-1.16-old/src/Makefile.am	2009-10-28 09:45:47.000000000 +0100
+++ ftrack-1.16/src/Makefile.am	2010-02-22 04:39:35.000000000 +0100
@@ -16,5 +16,5 @@ rntrack_LDFLAGS = -L$(srcdir)/smapi
 rntrack_LDADD = $(srcdir)/smapi/libsmapi.a
 
 INCLUDES = -I$(top_srcdir)/hpp -I$(srcdir)
-CFLAGS = $(CFLAGS) -pedantic -Wall -fno-rtti @PERL_CFLAGS@
+#CFLAGS = $(CFLAGS) -pedantic -Wall -fno-rtti @PERL_CFLAGS@
 #LFLAGS = $(LFLAGS) @PERL_LFLAGS@
diff -urp ftrack-1.16-old/src/Makefile.in ftrack-1.16/src/Makefile.in
--- ftrack-1.16-old/src/Makefile.in	2009-10-28 09:45:47.000000000 +0100
+++ ftrack-1.16/src/Makefile.in	2010-02-22 04:40:48.000000000 +0100
@@ -97,7 +97,8 @@ AUTOMAKE = @AUTOMAKE@
 AWK = @AWK@
 CC = @CC@
 CCDEPMODE = @CCDEPMODE@
-CFLAGS = $(CFLAGS) -pedantic -Wall -fno-rtti @PERL_CFLAGS@
+#CFLAGS = $(CFLAGS) -pedantic -Wall -fno-rtti @PERL_CFLAGS@
+CFLAGS = @CFLAGS@
 CPP = @CPP@
 CPPFLAGS = @CPPFLAGS@
 CXX = @CXX@
