--- Mail-SpamAssassin-4.0.0/sa-update.raw	2022-12-14 06:03:27.000000000 +0000
+++ Mail-SpamAssassin-trunk/sa-update.raw	2023-06-25 18:07:40.969999796 +0000
@@ -22,11 +22,11 @@ use warnings;
 use re 'taint';
 
 my $VERSION = 'svnunknown';
-if ('$Id: sa-update.raw 1900642 2022-05-07 06:01:02Z hege $' =~ ':') {
-  # Subversion keyword "$Id: sa-update.raw 1900642 2022-05-07 06:01:02Z hege $" has been successfully expanded.
+if ('$Id: sa-update.raw 1910601 2023-06-25 17:54:44Z hege $' =~ ':') {
+  # Subversion keyword "$Id: sa-update.raw 1910601 2023-06-25 17:54:44Z hege $" has been successfully expanded.
   # Doesn't happen with automated launchpad builds:
   # https://bugs.launchpad.net/launchpad/+bug/780916
-  $VERSION = &Mail::SpamAssassin::Version . ' / svn' . (split(/\s+/, '$Id: sa-update.raw 1900642 2022-05-07 06:01:02Z hege $'))[2];
+  $VERSION = &Mail::SpamAssassin::Version . ' / svn' . (split(/\s+/, '$Id: sa-update.raw 1910601 2023-06-25 17:54:44Z hege $'))[2];
 }
 
 my $PREFIX          = '@@PREFIX@@';             # substituted at 'make' time
@@ -1458,7 +1458,7 @@ sub do_dns_query {
       next if !$rr;  # no answer records, only rcode
       next if $rr->type ne $rr_type;
       # scalar context!
-      my $text = $rr->UNIVERSAL::can('txtdata') ? $rr->txtdata : $rr->rdatastr;
+      my $text = $rr->UNIVERSAL::can('txtdata') ? $rr->txtdata : $rr->rdstring;
       push(@result,$text)  if defined $text && $text ne '';
     }
     printf("DNS %s query: %s -> %s\n", $rr_type, $query, join(", ",@result))
@@ -2056,6 +2056,19 @@ configuration, based on channels.  The d
 I<updates.spamassassin.org>, which has updated rules since the previous
 release.
 
+NOTE: channel names are domain names, but DO NOT typically have any DNS
+records other than (maybe) NS records. There is a tree of records below that
+name which denote the SpamAssassin version and resolve that name to the 
+version number of the latest rules, e.g. to find the latest update
+version number for SpamAssassin v4.0.0:
+
+  $ host -t txt 0.0.4.updates.spamassassin.org 
+  0.0.4.updates.spamassassin.org is an alias for 3.3.3.updates.spamassassin.org.
+  3.3.3.updates.spamassassin.org descriptive text "1907730"
+
+That also illuminates the fact that the current ruleset is supposed to be 
+backward-compatible to v3.3.3. 
+
 Update archives are verified using GPG signatures by default.  If GPG is
 disabled (not recommended), file integrity is checked with SHA512 or SHA256
 checksums.
